### DISCUSSION TITLE
How reveal folders?

### DISCUSSION THREAD

**Discussion by SoheilaBagherzade** [elementary/code#1352] [Q&A] [answered]
![Screenshot (23)](https://github.com/elementary/code/assets/109869583/adb35997-572b-4136-93a4-e524458d0e6d)
Hello friends. I already created a series of folders in the codespace environment, but I hide them out of misplaced curiosity. Now, when I go to the path of those folders in Terminal, I can access the files inside, but I can't find the folders graphically in the left bar. If I create a new folder, it does not find them. I would be grateful if someone who is familiar with this environment can help me reveal the previous folders.

**Comment by davidmhewitt** [comment, marked as answer]
This GitHub page is for the code editor for elementary OS which is a desktop application on a Linux distribution.

Your screenshot shows Microsoft VSCode on the web which is not related or affiliated.

You will need to find the relevant support forum for this question, so I'll close this discussion.
